cmd/generate-manifests/main.go

```diff
@@ -0,0 +1,17 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package main
+
+import (
+	"os"
+
+	"github.com/gardener/cert-management/pkg/deployer"
+)
+
+func main() {
+	deployer.GenerateManifestsCommand(os.Args[1:], "")
```

Comment:
Seems like we don't need to pass the second argument at all?!

Reply:
I have kept the subcommand in the cert-controller-manager image, see [cmd/cert-controller-manager/main.go line 76](https://github.com/gardener/cert-management/blob/enh/deployer/cmd/cert-controller-manager/main.go#L76).
That's the reason for this second argument.